themes/default/content/docs/intro/languages/javascript.md

```diff
@@ -14,6 +14,14 @@ aliases: ["/docs/reference/javascript/"]
 
 Pulumi supports writing your infrastructure as code in any JavaScript language running on Node.js using any of the [Current, Active and Maintenance LTS versions](https://nodejs.org/en/about/releases/).
 
+{{% notes type="warning" %}}
+Pulumi currently does not support NodeJS 19.2 and higher due to internal V8 changes that our function serialization doesn't currently handle.
```

Comment:
Is it actually true that we don't support it at all, or only that closure serialization does not work on 19.2?

Reply:
It's just closure serialisation, but we're not testing against 19.2 untill we can split off all the tests that use closure serialisation from everything else (and it's a lot because so many of the node tests use dynamic providers as part of the tests)

Reply:
I wonder if it would help if I flip these sentences around a bit? Say that we expect 19.2 to work except for function serialization, then link the issue, then comment that we're not currently testing 19.2 so we don't consider it fully supported?

Reply:
I like that additional bit of nuance to this note - it feels like it re-centers the issue on function serialization and support guarantees re: testing rather than sounding like we don't think people should use 19,2 with Pulumi.

+1